app/controllers/verify/sessions_controller.rb

```diff
@@ -20,6 +19,18 @@ def new
     end
 
     def create
+      result = idv_form.submit(profile_params)
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED_FORM, result.to_h)
+
+      if result.success?
+        submit_idv_job
+        redirect_to verify_session_result_path
+      else
+        process_failure
+      end
+    end
+
+    def show
```

Comment:
I thought we were sticking to the Rails convention of using only built-in action names as much as possible (`index`, `edit`, `show`, `create`, etc etc). To me, this is showing the result of the vendor result, which sometimes is in progress (hence rendering a spinner).

Re: when the vendor times out, I figured that incrementing the count was acceptable for a few reasons:
1. if a vendor is permanently down, we should not continue to spam them with requests and waste their time
2. if the request did get to the vendor, then it counts towards our limit on their side anyways

I don't think the experience of timing out is ideal, but I think that treating timeout errors as negative responses is acceptable.

Re messaging: I'll look into adding a "our request to verify your information timed out" error message.